packages/core/admin/ee/admin/pages/SettingsPage/pages/ApplicationInfosPage/components/AdminSeatInfo/index.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { useIntl } from 'react-intl';
+import { Flex, Tooltip, Icon, GridItem, Typography, Stack } from '@strapi/design-system';
+import { Link } from '@strapi/design-system/v2/Link';
+import { ExternalLink, ExclamationMarkCircle } from '@strapi/icons';
+import { pxToRem } from '@strapi/helper-plugin';
+import { useLicenseLimits } from '../../../../../../hooks';
+
+const BILLING_STRAPI_CLOUD_URL = 'https://cloud.strapi.io/profile/billing';
+const BILLING_SELF_HOSTED_URL = 'https://share.hsforms.com/1WhxtbTkJSUmfqqEuv4pwuA43qp4';
```

Comment:
Adding billing URLs in the codebase directly is not really a viable long-term solution. That's not something we'll be able to change later if needed.

I did not follow the internal discussions about this topic, but it feels off.

Reply:
I agree and had the same thought when I saw it, but didn't want to request it because I don't know what has been discussed. Is the strapi link shortener capable of changing target links? If so I guess that would be preferred, so that we have to chance anytime to change these targets.

Reply:
What can we do here?

Reply:
Check if the strapi link shortener could be useful here I'd say - if not, find out if we have another way of creating stable links we can ship but be able to change the target?

Reply:
We might have a solution here, currently being discussed 👍

Reply:
Btw, what's Strapi link shortener?

Reply:
The [strp.cc](https://t.co/Sqlc6OA0Ph) thing, but I don't know whom you have to ask about that. Maybe Nastia?

Reply:
Gotcha 👍 we're actually thinking of launching a small app that would have only 1 url for these things and determine what to do based on parameters, such as subscriptionId